src/main/resources/db/migration/V0001__create_user.sql

```diff
@@ -1 +1,26 @@
-create table users (id serial primary key);
\ No newline at end of file
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+--texts
+
+CREATE TABLE texts(
+	id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
+	text_content varchar(5120),
+	text_link varchar(400),
+	text_language varchar(15) NOT NULL,
+	date_added timestamp(6),
+	user_id uuid NOT NULL,
+	UNIQUE(user_id)
```

Comment:
are you sure the `user_id` is unique ? The table order must be reversed as you'll need to specify the `foreign key` constraint on the table